### ISSUE TITLE
How about build for libretro

### ISSUE THREAD

**Issue by litaniast** [captainys/TOWNSEMU#2]
If Android and Raspberry Pi users can play FM TOWNS games through libretro, it would be exciting!

**Comment by captainys** [comment]
Hi Litaniast,

It should be possible.  I’m not sure about Android, but I am fully intending to make it cross-platform.  But Tsugaru is pretty demanding.  It may not perform well on Raspberry Pi.

Thanks!

Soji

**Comment by emptyinside666** [comment]
Works beautifully on PC but wondering what the status is for a Retropie build.  Keep trying to Make but am confused as the instructions are minimal for noobish folks and prefer not to run exe through Wine.

**Comment by BenoitAdam94** [comment]
Yeah a Retroarch/libretro core would be amazing because this emulator is great. Even if it's a basic one that only work with few games

I think we need to ping libretro cores dev
